guides/source/action_mailbox_basics.md

```diff
@@ -265,9 +267,15 @@ https://actionmailbox:PASSWORD@example.com/rails/action_mailbox/sendgrid/inbound
 
 NOTE: When configuring your SendGrid Inbound Parse webhook, be sure to check the box labeled **“Post the raw, full MIME message.”** Action Mailbox needs the raw MIME message to work.
 
-## Examples
+## Processing Incoming Email
+
+Processing incoming emails usually entails using the email content to create models, update views, queue background work, etc. in your Rails application.
+
+Before you can start processing incoming emails, you'll need to setup Action Mailbox routing and create mailboxes.
+
+### Configure routing
```

Comment:
For the guide headings we try to:
> capitalize all words except for prepositions, conjunctions, internal articles, and forms of the verb "to be"

https://guides.rubyonrails.org/ruby_on_rails_guides_guidelines.html#headings
```suggestion
### Configure Routing
```

Reply:
Thanks @p8